api-docs/rst/dev-guide/concepts.rst

```diff
@@ -61,40 +68,65 @@ Scaling groups
-| Launch Configuration | Creates a blueprint for how new servers will be created. The launch configuration specifies what type of server image will be started on launch, what flavor the new server is, and which cloud load balancer or RackConnect v3 load balancer pool the new server connects to.                                        |
-|                      | Note: The launchConfiguration uses the admin user to scale up, usually the first admin user found on the tenant. Only that particular admin user's SSH key pair names can be used in the launchConfiguration.                                                                                                         |
-|                      | Note: The launchConfiguration update operation overwrites all launchConfiguration settings.                                                                                                                                                                                                                           |
-+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
-The launch configuration specifies the launch type along with server and load balancer configuration for the components to start. Most launch configurations have both a server and a load balancer (can be RackConnect v3) configured as shown in the `Launch configuration examples`_ .
++----------------------+------------------------------------------------------+
+| Configuration        | Description                                          |
++======================+======================================================+
+| Group Configuration  | Outlines the basic elements of the Autoscale         |
+|                      | configuration. The group configuration manages how   |
+|                      | many servers or stacks can participate in the        |
+|                      | scaling group. It sets a minimum and maximum limit   |
+|                      | for the number of entities that can be used in the   |
+|                      | scaling process. It also specifies information       |
+|                      | related to load balancers.                           |
++----------------------+------------------------------------------------------+
+| Launch Configuration | Creates a blueprint for how new servers or stacks    |
+|                      | will be created.                                     |
+|                      |                                                      |
```

Comment:
Creates a blueprint for the number of new servers or stacks that can be created.